cephfs/admin/response.go

```diff
@@ -0,0 +1,101 @@
+// +build !luminous,!mimic
+
+package admin
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+)
+
+var (
+	// ErrStatusNotEmpty may be returned if a call should not have a status
+	// string set but one is.
+	ErrStatusNotEmpty = errors.New("response status not empty")
+	// ErrBodyNotEmpty may be returned if a call should have an empty body but
+	// a body value is present.
+	ErrBodyNotEmpty = errors.New("responase body not empty")
```

Comment:
nit: response

Also the comit message has "handling" twice.

Reply:
good eye, thanks! Will fix.